### ISSUE TITLE
Love / Copy notification WILL NOT GO AWAY!

### ISSUE THREAD

**Issue by finsaah** [simple-last-fm-scrobbler/sls#536]
As title - what's going on here?  Please make it go away.

**Comment by inverse** [comment]
Yeah this one is quite annoying. We should have the option to disable it.

**Comment by a93h** [comment]
Please choose to disable "Active App Notification" in power settings aka "Options"

**Comment by finsaah** [comment]
"warning may skip scrobbles" seeing as that's the whole point of the app, it seems a bit odd to have scrobbles linked to this option

**Comment by a93h** [comment]
I assumed it was obvious, as I would not add notifications if they weren't necessary.
If disabled, some battery optimized devices may skip scrobbles.
If you permitted the app not to be battery optimized it should work.

**Comment by inverse** [comment]
When I disable this the notification still is there until the song changes. Thanks for the heads up though :+1:

**Comment by sn0skier** [comment]
This notification is persistent for me even when there music app I was using had been paused and exited (by which I mean the music app notification has been swiped). Is this notification supposed to be persistent even after closing the associated music app? So the notification will be permanent until I restart the phone?

I'm using GoneMad music player and simple scrobbler version 1.7.0 on Android 9.

**Comment by a93h** [comment]
@sn0skier 

Please choose to disable "Active App Notification" in power settings aka "Options". After changing a song it will disable the notification.

Have you attempted this friend?

**Comment by sn0skier** [comment]
> @sn0skier
> 
> Please choose to disable "Active App Notification" in power settings aka "Options". After changing a song it will disable the notification.
> 
> Have you attempted this friend?

Yes, I currently have this setting disabled, but I would like to enable it, but I only want to see the notification when a song is playing, not all the time even when no song is playing. If this isn't possible I guess that's OK because scrobbling seems to working even with this disabled. Thank you for your help.

**Comment by a93h** [comment]
This following apk should function well.
Not 100% certain it will fix the problem, but it's worth a try.
[app-debug.zip](https://github.com/simple-last-fm-scrobbler/sls/files/3905015/app-debug.zip)

**Comment by a93h** [comment]
Remember to disable Active App Notification

**Comment by sn0skier** [comment]
> This following apk should function well.
> Not 100% certain it will fix the problem, but it's worth a try.
> [app-debug.zip](https://github.com/simple-last-fm-scrobbler/sls/files/3905015/app-debug.zip)

Sorry, but I don't have time to debug this right now and I don't know when it will. As I stated above, scrobbling is working correctly for me now even with the notification disabled, so fixing this isn't a priority for me right now. Thank you for your help and I'm sorry I won't be able to help debug this. I'm sure I'm not the only one that wanted this fixed though, so hopefully someone else will be able to confirm the fix.

**Comment by pwd-github** [comment]
No matter what I do, I can't remove this persistent notification. I've tried on my LG G6 and my Pixel 4XL. I'm getting to the point that I just want to give up on this once great and simple app.

**Comment by ShadowMax** [comment]
Same problem.
https://github.com/simple-last-fm-scrobbler/sls/issues/597#issue-945288033 get it
